### ISSUE TITLE
SPRINT 42 PRIORITIES

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#20421]
## Instructions for first few sprints using this priority document:

- This is a living document that everyone is empowered to keep updated.  
- If you complete a task, please come here to check if off.  If you add a new issue or run across something that needs to be discussed in the next grooming session, please include add it to Discussion Topics/Grooming.

## Sprint 42 Priorities

1.  Implement Google Analytics data layer event code for Yellow Ribbon (remaining from Sprint 41) - Complete on our end; Jon said on 3/8/2021 he will look at it early this week.
2.  ~Brainstorm options for matching different Find a VA Form search variations against existing form metadata to avoid creating new Lighthouse, CMS dependencies.~
3.   Monitor bugs CMS, Lighthouse, VA Forms Mgrs are working on
4. Yellow Ribbon event verification/Domo Dashboard request (blocked by 1)
5. ~Complete the [GH read-me](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/find-va-forms-redesign/top-forms.md) which captures the top searched Forms and indicate if they have an online tool link, related forms May have to call the API from Postman. - Zach will write a request that I can drop into Postman (?)~
6. **CANCELLED** ~Find a VA Form: Update Search Results according to Style Guide; increase visibility of CTA links #20295~
7. ~Find a VA Form: Familiarize Cassandra of plan to display form "teaser" info in search results #20267~

<hr>

#### Zach
- [x] Brainstorm options for matching different Find a VA Form search variations against existing form metadata to avoid creating new Lighthouse, CMS dependencies.
- [ ] #20651 Find a VA Form: Explore technical feasibility of transforming search terms
- [x] Find a VA Form: Update Search Results according to Style Guide; increase visibility of CTA links #20295
- [ ] Find a VA Form: Dev/Staging POC to display form "teaser" info in search results #20267 - PAUSE - Marci will set up a meeting with Ryan Thurlwell to discuss new mock-ups he's done to standardize the form elements.   Adding teaser text would be a second iteration on top of that.
- [x] Define work needed from Lighthouse to enable search string matching improvements **(need to create an issue for this)**
- [x] Diagram testing/deployment process
- [x] Postman query Marci can use to get a list of all forms & their attributes

#### Cassandra (NEW!) 

- [x] Introduce to team
- [x] Overview of upcoming work requiring design, research - https://github.com/department-of-veterans-affairs/va.gov-team/issues/20267
- YR user testing
- Find a VA form user testing

#### John
- [ ] TBD

#### Marci
- [x] Diagram testing/deployment process
-  [ ] #20651 Find a VA Form: Explore technical feasibility of transforming search terms
- [ ] CMS dev training / request Prod credentials
- [ ] Review Prod CMS for any new forms needing details pages
- [x] Check Find a VA form SF forms for broken links; check CMS to see if issues originate there.
- [x] GitHub training Mar 4
- [ ] Organize product folders in GitHub
- [x] Create GitHub issues for tasks needed to improve search string matching #19601
- [x] Monitor bugs Lighthouse is working on
- [x] Monitor Analytics KPI dashboard request for Decision Tools Team - Find a VA Form #19689 

<hr>

#### Zach
- [  Analytics Review on Staging  #18234  **Complete on our end; Jon said on 3/8/2021 he will look at it early this week.**
- [ ] Re-deploy search tips to Prod post-Analytics review  #18234  **Complete on our end; Jon said on 3/8/2021 he will look at it early this week.**

#### John
- [ ] TBD

#### Marci
- [ ] Verify new events, create Domo dashboard/reports request for Yellow Ribbon  3/5/2021 - pinged Jon W on Mon & again Fri for status update.  We're holding pending his review.  Complete on our end; Jon said on 3/8/2021 he will look at it early this week.

### 🛠️ Technical Debt / A11y Issues
**Our goal is to include 1-2 tech debt issues into each sprint.**

#### Zach
- [ ] sitewide 508-defect-2 [COGNITION, SCREENREADER]: Pagination SHOULD use meaningful landmarks and aria techniques #18398  Note: Josh K is OOO
- [ ] sitewide 508-defect-2 [COGNITION, SCREENREADER]: Heading SHOULD read as a single item #18389
<hr>  - Note: Josh K is OOO

#### CMS
None at this time

#### Lighthouse 
- [x] CMS Backlog:  department-of-veterans-affairs/va.gov-cms#4083 | Original issue #18920 
   - 2021/02/24 Per Nichole, this didn't fit into the last sprint, but is prioritized for the current one.
  
- [ ] [Bug] Find a VA form search is showing duplicate listing of VA Form 10182 #17484 
    - 2021/03/05 - Mark V and Zach are collaborating on a solution
    - 2021/02/26 - Asked for this issue to be lower priority than #19561
    - 2021/02/26 - Met with Lighthouse, Steve Wirt, Zach to discuss.
 
- [ ] [BUG] Find a VA Form: "Related to" information missing on some results #19561
   - 2021/03/04 - Mark V and Zach are collaborating on a solution.  Mark has made his changes & Zach will need to make changes to his code to consume the new data.
   - 2021/02/26 - Asked for this issue to be higher priority than #17484
   - 2021/02/24 Per Nichole, prioritized for sprint that is now starting.

#### VA Forms Managers

- [ ] Find a VA Form [BUG] - Invalid character in VA DB PDF URL is causing "not found" error. #18926

<hr>

**Comment by zacharymorel** [comment]
@MarciMcGuireGCIO here are the URLS regarding the query to get all forms from the lighthouse API:

**Staging** GET: https://staging-api.va.gov/v0/forms?query=
**PROD** GET: https://api.va.gov/v0/forms?query=

**Comment by MarciMcGuireGCIO** [comment]
Thank you! 😄

**Comment by zacharymorel** [comment]
@MarciMcGuireGCIO @johnhashva here's the technical doc regarding _work needed from Lighthouse to enable search string matching improvements_.
https://github.com/department-of-veterans-affairs/va.gov-team/issues/20517

**Comment by MarciMcGuireGCIO** [comment]
Thank you, @zacharymorel !
